CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# How to contribute?
+
+First of all, thanks for taking the time to contribute! :tada::+1:
+
+Your contributions are important to make this project and event successful. 
+You can report mistakes and errors, propose a project, offer dataset and resources or create more content. 
+
+Whatever is your background, there is a way to contribute on this [GitHub repository](https://github.com/virtual-biohackathons/covid-19-bh20), 
+on our [mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/virtual-biohackathon)
+or other [related projects listed on our wiki page](https://github.com/virtual-biohackathons/covid-19-bh20/wiki).
+
+You can raise any issue relevant to this project/event publicly in the mailing list (google group), we will address and/or assess your change proposal as promptly as we
+can to make your participation easier.
+
+We have a [Code of Conduct](https://github.com/virtual-biohackathons/covid-19-bh20/blob/master/CODE_OF_CONDUCT.md) that applies to all the activities and projects involved in this event.
+For any organisation related queries or concern, you can directly reach out to the organisers Leyla Garcia (email: ljgarciaco@gmail.com), Pjotr Prins (email: pjotr.public708@thebird.nl) 
+and  Tazro Ohta (email: t.ohta@dbcls.rois.ac.jp). 
+
+**ADD MISSING DETAILS**
```

Comment:
With such a large group of participants, we are encouraging a bottom-up approach, where groups are formed around projects/topics and work (kind of) independently from others. If you need any information from another group/topic, you can always reach people via email or slack (please note that as the hacking starts slack might become more interactive, you can also directly ping people there, be patient please and re-ping if needed or look for an alternative contact). Every group/topic will have leader/coordinator/facilitator, please check the wiki to get more information (as we get close to the hacking week, we expect wiki to change quickly and people to add more group/topic details there). We also expect group/topic leaders/coordinators/facilitators to provide a quick (daily if possible) summary wia the wiki so everybody can get an idea on it.